proposed/cache.md

```diff
@@ -227,21 +235,31 @@ interface CacheItemInterface
     public function exists();
 
     /**
-     * Sets the expiration for this cache item.
+     * Sets the expiration time this cache item.
```

Comment:
Not sure if this is a correct sentence. It at least does not make it clear what the method is expected to do to me.